crates/sui-graphql-client/src/lib.rs

```diff
@@ -765,6 +965,63 @@ impl Client {
         }
     }
 
+    /// Return the contents' JSON of an object that is a Move object.
+    ///
+    /// If the object does not exist (e.g., due to prunning), this will return `Ok(None)`.
```

Comment:
```suggestion
    /// If the object does not exist (e.g., due to pruning), this will return `Ok(None)`.
```

Also -- what about if it was deleted or wrapped?